"GitHub is not a constructor" error with v2.2.1

Hi, we are getting the error

`##[error]github.GitHub is not a constructor`

across multiple repos with the latest release. v2.2.0 still works good

--- a/.github/workflows/pull-request.yml
+++ b/.github/workflows/pull-request.yml
@@ -12,7 +12,10 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: aslafy-z/conventional-pr-title-action@master
+      - name: Check out repository
+        uses: actions/checkout@v2
+
+      - uses: ./
         with:
           preset: conventional-changelog-angular@^5.0.6
         env:

--- a/package.json
+++ b/package.json
@@ -23,7 +23,7 @@
   "homepage": "https://github.com/aslafy-z/conventional-pr-title-action#readme",
   "dependencies": {
     "@actions/core": "^1.2.5",
-    "@actions/github": "^4.0.0",
+    "@actions/github": "^2.0.0",
     "conventional-commit-types": "^3.0.0",
     "conventional-commits-parser": "^3.1.0",
     "npm-package-arg": "^8.0.1"

